src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -20,8 +20,13 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
 
     val timestamp = new SimpleDateFormat("YYYY-MM-dd HH:mm:ss,SS").format(new Date())
 
+    val ansiColor = Option(System.getProperty("RAINBOW_UUID")) match {
+      case Some(_) => "UUID\\((.*?)\\)".r.findFirstMatchIn(event.getFormattedMessage).map(s => (Math.abs(17 * s.group(1).map(_.toInt).product) % 209) + 22).getOrElse(2)
+      case None => 2
+    }
+
     val highlightedMessage = event.getFormattedMessage
-      .replaceAll("UUID\\((.*?)\\)", TerminalUtil.highlight(2, "$1"))
+      .replaceAll("UUID\\((.*?)\\)", TerminalUtil.highlight(ansiColor, "$1"))
```

Comment:
A wild terminal layout change has appeared!

Is this a duplicate of Scott's branch or genuinely meant to be here?